.dockerignore

```diff
@@ -0,0 +1,8 @@
+/lib/
+/classes/
+/targets/
+target/
+/.bundle/
+/resources/puppetlabs/puppetserver/*.class
+/tmp/
+/dev-resources/i18n/bin
```

Comment:
Not sure in this case, but it may be easier to invert these .dockerignore files so we ignore everything and then only add in the things we need, like we did for puppetdb: https://github.com/puppetlabs/puppetdb/blob/master/.dockerignore

That way you don't accidentally get a Gemfile.lock sneaking into your docker build context and changing your behavior, or some other random file that gets added to the repo but not added to .dockerignore.

Not a big deal, just mentioning the different approaches.

Reply:
ahh, good call @nwolfe, will update